models/public/device_support.md

```diff
@@ -166,3 +166,4 @@
 | yolo-v3-tiny-tf | YES | YES | YES |
 | yolo-v4-tf | YES | YES | YES |
 | yolo-v4-tiny-tf | YES | YES |    |
+| yolof | YES |   |    |
```

Comment:
do you not verify or know that model work only on CPU?

Reply:
seems model works on GPU